src/metabase/query_processor/streaming/xlsx.clj

```diff
@@ -365,12 +365,7 @@
 ;; `metabase.server.middleware`.
 (defmethod set-cell! Object
   [^Cell cell value _styles _typed-styles]
-  ;; stick the object in a JSON map and encode it, which will force conversion to a string. Then unparse that JSON and
-  ;; use the resulting value as the cell's new String value.  There might be some more efficient way of doing this but
-  ;; I'm not sure what it is.
-  (.setCellValue cell (str (-> (json/generate-string {:v value})
-                               (json/parse-string keyword)
-                               :v))))
+  (.setCellValue cell (json/generate-string value)))
 
```

Comment:
I don't really understand why this is written this way. @camsaul maybe you know?
I think it's better to just use the json encoded string as the cell value. 

This way, we export xlsx files with values that look like json, matching the app. Before this change, a json-looking column is downloaded and looks like clojure edn, which most people think is broken json.

Reply:
Honestly this was 4 years ago and I really don't remember what my exact thinking was, but I'm guessing it was probably that if we had no other encoding method for something the fallback (`Object` method) would be to write it out as a some as a string representation. By using `json/generate-string` we could leverage our Cheshire methods for converting various Clojure data types to something more universally understood. I don't really remember why I have it reading it back to EDN tho. I think your change makes a lot of sense